gprofiler/metadata/application_identifiers.py

```diff
@@ -69,10 +69,14 @@ def _append_python_module_to_proc_wd(process: Process, module: str) -> str:
     if not module.endswith(".py"):
         module = module.replace(".", "/") + ".py"
 
-    if os.path.isabs(module):
-        return module
+    return _append_file_to_proc_wd(process, module)
 
-    return os.path.realpath(os.path.join(process.cwd(), module))
+
+def _append_file_to_proc_wd(process: Process, file_path: str) -> str:
+    if os.path.isabs(file_path):
+        return file_path
+
+    return os.path.realpath(os.path.join(process.cwd(), file_path))
```

Comment:
`realpath` reads links but the path is in the process's namespace... How can it run?

Reply:
```python
In [2]: os.path.realpath("/not/a/real/./../path")
Out[2]: '/not/a/path'
```
🤷🏽

Reply:
The docs say it reads links. Sigh